src/partials/hero.html

```diff
@@ -1,8 +1,10 @@
 <header class="page-header">
   <div class="container page-header__container">
```

Comment:
missing closing tag

Reply:
the </div> closing tag is in the 59th line